packages/samples/mvp-dapp/src/main.ts

```diff
@@ -29,7 +29,7 @@ async function bootstrap() {
 
   const koaRouterAdapter = new KoaRouterAdapter(cor)
 
-  app.use(koaRouterAdapter.routes()).use(koaRouterAdapter.allowedMethods())
+  app.use(koaRouterAdapter.routes())
```

Comment:
`Methods` is managed by `kuaiRouter`, so there is no need to use `allowedMethods`.